### DISCUSSION TITLE
Room Cleaning

### DISCUSSION THREAD

**Discussion by itsunreal1** [Tasshack/dreame-vacuum#293] [Q&A] [answered]
Hello, I am quite new in HomeAssistant but I managed to install the dreame-vacuum adapter. The basic functions works.
But how can I start a room cleaning? Also would be great if I can set some parameters. For example: Start Cleaning room Bathroom1, Mopping+Suction, Suction Level 4.

**Comment by Tasshack** [comment, marked as answer]
Most of the information you need is already in the repo. This is not a support page so nobody will help you here, you have to read the info.md and google the information you have gathered.
